src/main/java/de/uhd/ifi/se/decision/management/jira/consistency/implementation/TracingCIP.java

```diff
@@ -0,0 +1,81 @@
+package de.uhd.ifi.se.decision.management.jira.consistency.implementation;
+
+import com.atlassian.jira.component.ComponentAccessor;
+import com.atlassian.jira.issue.Issue;
+import com.atlassian.jira.issue.link.IssueLinkManager;
+import de.uhd.ifi.se.decision.management.jira.consistency.ContextInformationProvider;
+
+import java.util.*;
+
+public class TracingCIP implements ContextInformationProvider {
+	private String id = "TracingCIP_BFS";
+	private String name = "TracingCIP";
+	IssueLinkManager issueLinkManager = ComponentAccessor.getIssueLinkManager();
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use explicit scoping instead of the default package private level](https://app.codacy.com/manual/UHD/cures-condec-jira/pullRequest?prid=5455522)